docs/concepts/work-pools.md

```diff
@@ -10,7 +10,7 @@ tags:
     - concurrency limits
 ---
 
-# Agents, Work Pools, & Work Queues
+# Agents, Workers Work Pools, & Work Queues
```

Comment:
```suggestion
# Work Pools, Workers, & Agents
```

Reply:
Lets use this as an opportunity to de-emphasize work queues as a top level concept and emphasize the work pool as the primary concept.